### PR TITLE
New version: CologneCodeCompany.XYplorer version 26.20.0200

### PR DESCRIPTION
**Pull request by The-Running-Dev**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162875)

### CODE REVIEW

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200
```
Comment (Trenly) [inline manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.installer.yaml:5]:
```suggestion
PackageVersion: 26.20.0203
```
Version should match what is written to control panel. After changing this, you will need to update the name of the version folder

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200
```
Comment (Trenly) [inline manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 26.20.0203
```

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200
+PackageLocale: en-US
+Publisher: Cologne Code Company
```
Comment (Trenly) [inline manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml:7]:
```suggestion
Publisher: Donald Lessau, Cologne Code Company
```
This should match control panel

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: CologneCodeCompany.XYplorer
+PackageVersion: 26.20.0200
```
Comment (Trenly) [inline manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.yaml:5]:
```suggestion
PackageVersion: 26.20.0203
```

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml (hunk 3 above)
Comment (The-Running-Dev) [reply manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml:7]: What control panel?

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml (hunk 2 above)
Comment (The-Running-Dev) [reply manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.locale.en-US.yaml:5]: Same issue as above, where is .0203 coming from?

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.yaml (hunk 4 above)
Comment (The-Running-Dev) [reply manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.yaml:5]: Same issue as above, where is .0203 coming from?

File: manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/c/CologneCodeCompany/XYplorer/26.20.0200/CologneCodeCompany.XYplorer.installer.yaml:5]: What's meant here is really the Registry. Both the Control Panel's Add and remove programs page (aka ARP) and the Apps and features Settings page (aka ANF) display the Registry data for installed applications. And this data can be retrieved with other means, including the Registry Editor. 

Does the application display a different version number? 

Sometimes, an application will be written to display one version number, or the version number in one format - while the application's installer is configured to write a different version number, or the version number in another format, to the Registry. When this happens, the usual solution is adding a `DisplayVersion` to match the Registry data. This way, the package manager can display a version number that matches the application, by matching repo data with Registry data. 

I haven't installed the application, so I'm not sure if this is occurring here. But it sounds like the application might display `26.20.0200`, while the installer writes `26.20.0203` to the Registry. Is this the case?

**Comment by The-Running-Dev** [comment]
Abandoning this in favor of re-creating a new one with the correct version number.

**Pull request closed without merging**